packages/react-keytips/package.json

```diff
@@ -13,5 +10,11 @@
     "@types/react-dom": ">=16.8.0 <19.0.0",
     "react": ">=16.8.0 <19.0.0",
     "react-dom": ">=16.8.0 <19.0.0"
+  },
+  "dependencies": {
+    "@fluentui/react-jsx-runtime": ">=9.0.29 < 10.0.0",
+    "@fluentui/react-utilities": ">=9.16.0 < 10.0.0",
+    "@fluentui/react-positioning": "^9.16.0",
+    "@fluentui/keyboard-keys": "~9.0.6"
```

Comment:
which of these apis that this library is using are not part of react-components suite ?

if api's are accessible via rc suite those sub-packages should not be specified as dependencies.

**what needs to change** are the versioning ranges, `gte` etc is supposed to be used only for peerDependencies, anything else needs to use `~` or `^` depending on the used package semver policy

cc @dmytrokirpa ( we talked about this inconsistency in whole contrib - not sure if you managed to create that issue yet )

Reply:
can you please open issue on this topic in core repo ?